components/AppNav.tsx

```diff
@@ -17,47 +17,47 @@ type AuthLinkProps = {
   session: any
 }
 
+type NavItem = NavLinkProps & { name: string }
+
 const dropdownMenuItems = [
   { title: 'Lessons', path: '/admin/lessons' },
   { title: 'Users', path: '/admin/users' },
   { title: 'Alerts', path: '/admin/alerts' }
 ]
+const navItems: NavItem[] = [
+  { path: '/curriculum', name: 'Curriculum' },
+  {
+    path: 'https://github.com/garageScript/c0d3-app',
+    name: 'Repo',
+    external: true
+  },
+  {
+    path:
+      'https://www.notion.so/Table-of-Contents-a83980f81560429faca3821a9af8a5e2',
+    name: 'Journey',
+    external: true
+  },
+  { path: 'https://chat.c0d3.com', name: 'Help', external: true },
+  { path: '/contributors', name: 'Contributors' }
+]
 
 const NavBar: React.FC<AuthLinkProps> = ({ session }) => {
   const isAdmin = _.get(session, 'user.isAdmin', '')
+  const location = window && '/' + window.location.pathname.split('/')[1]
```

Comment:
We should stick to just using `useRouter` path related things in order to maintain codebase consistency.

Reply:
Using `router` turns this pr into #509. Considering that it took more than 12 days to review it I think people don't really like to review 40 files at once.

Reply:
It's unfortunate that it may take longer to get reviewed, but that factor is not a good enough reason to decrease the quality of the codebase

Reply:
Well, my intend was to fix highlighting with this pr and add router, router mocks and etc with the next pr.
Also, we are already using `window.location` for changing routes in some cases.

Reply:
> Also, we are already using `window.location` for changing routes in some cases.

That needs to be fixed as well, changing routes without the router forces the browser to refresh instead of just modifying the DOM.

Reply:
> Well, my intend was to fix highlighting with this pr and add router, router mocks and etc with the next pr.

Ah, I see! It makes sense that this is its own separate PR then. Can you make separate issues for each of the problems you are fixing, and tag the issues you are tackling in each PR description? Like for example putting a `closes #33` in the PR description. That will provide us a more organized way of letting us know which specific issue each PR is handling. Tagging issues in your PR description will also automagically close those tagged issues as well, which is really convenient:
https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue

Furthermore, each commit message should have an issue number(s) tagged too -- it's in our engineering practices:
https://github.com/garageScript/c0d3-app/wiki/Engineering-Practices

If I don't see the issue number of `changing window.location to router` being in the PR description, then I'll know the PR doesn't have the intention of doing that. And then maybe to add additional clarity for the reviewer, you can note in your PR description that are not going to change `window.location` to `router` in this PR, but that you've already made an issue for it so it will be addressed in a later PR.